### ISSUE TITLE
Авторизация

### ISSUE THREAD

**Issue by AlexPenkin** [mdnsk/leo-translate#34]
Не могу найти настройки авторизации. Не могу войти и соответственно сменить пользователя \ выйти. Из за этого не могу добавлять слова в словарь. Может плохо искал?

**Comment by alexdrupal** [comment]
У меня решилось тем, что я открыл еще одну вкладку браузера и там открыл сайт и залогинился. После этого смог добавить слова в словарь через расширение. Сайт держать все время открытым не обязательно, достаточно просто залогиниться.

**Comment by alok1111** [comment]
В последней версии 1.17.4 без авторизации не переводит. Так должно быть?

**Comment by mdnsk** [comment]
> В последней версии 1.17.4 без авторизации не переводит. Так должно быть?

Да. Такое ограничение ввели разработчики LinguaLeo. Теперь доступ к API есть только у авторизованных пользователей.

**Comment by vlod11** [comment]
Еще, как вариант, можно просто перелогинится
